packages/backend/src/modules/task/task.controller.ts

```diff
@@ -32,6 +32,16 @@ class TaskController {
     const taskDto: TaskDTO = this._taskMapper.mapToDto(task)
     return res.status(201).send(taskDto)
   }
+
+  async updateById(req: express.Request, res: express.Response) {
+    const updateTaskCommand: UpdateTaskCommand = req.body
+    const updatedById: Task = await this._taskService.updateById(
```

Comment:
The thing that will be stored here is an updatedTask 😄

Reply:
Done